Doc/library/ensurepip.rst

```diff
@@ -36,6 +36,7 @@ when creating a virtual environment) or after explicitly uninstalling
    :pep:`453`: Explicit bootstrapping of pip in Python installations
       The original rationale and specification for this module.
 
+.. include:: ../includes/wasm-notavail.rst
```

Comment:
I don't have a super strong feeling here, but maybe better to put the availability note right above the ``seealso`` instead of below? It feels a little unexpected to not have the "See Also" be the last element in the introduction, but maybe that's just me.